frontend/cypress/integration/featureFiles/services.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Kiali Services page
+
+  User opens the Services page and sees the bookinfo namespaces
+
+  Background:
+    Given user is at administrator perspective
+    And user opens the "services" page
+    And user selects the "bookinfo" namespace in the NamespaceSelector
+    And Kiali is done loading
+
+  @services-page
+  Scenario: See a table with correct info
+    Then user sees a table with headings
+      | Health | Name | Namespace | Labels | Configuration | Details |
+    And the "productpage" row is visible
+    And the health column on the productpage row has a health icon
```

Comment:
I think in this line the "productpage" should be passed as parameter instead of hardcoding:

![image](https://user-images.githubusercontent.com/1662329/165250940-7a1edfcf-3a13-491d-bc5f-42eabbef63bc.png)